test/middleware/ErrorHandler.test.ts

```diff
@@ -25,7 +26,7 @@ describe('Error Handler Middleware', () => {
             await request(app).get(FAKE_PAGE_URI)
                 .expect(response => {
                     expect(response.status).to.be.equal(NOT_FOUND);
-                    expect(response.text).to.contain(pageHeading);
+                    expect(response.text).to.contain(notFoundPageHeading);
```

Comment:
i feel we could expand on these expectations, i.e. check the response contains the email address and href